server/controllers/AuthController.js

```diff
@@ -112,4 +111,56 @@ export default class AuthController {
       return responseMessage(response, 500, { message: error.message });
     }
   }
+
+/**
+ *
+ *
+ * @static
+ * @param {*} request
+ * @param {*} response
+ * @returns {object} - response object
+ * @memberof AuthController
+ */
+  static async verifyUserbyEmail(request, response) {
+    const { email } = request.body;
+    try {
+      const user = await User.findOne({ where: { email } });
```

Comment:
use the helper function

Reply:
this seems a simpler its a single line, using the finduser helper will be joining tables and be providing data not needed